test/jellyfish_web/integration/server_socket_test.exs

```diff
@@ -217,4 +244,18 @@ defmodule JellyfishWeb.Integration.ServerSocketTest do
       content: {:auth_request, %PeerMessage.AuthRequest{token: token}}
     })
   end
+
+  defp trigger_notification(conn) do
+    server_api_token = Application.fetch_env!(:jellyfish, :server_api_token)
+    {room_id, _peer_id, peer_token, conn} = add_room_and_peer(conn, server_api_token)
+
+    {:ok, peer_ws} = WS.start_link("ws://127.0.0.1:#{@port}/socket/peer/websocket", :peer)
+    auth_request = peer_auth_request(peer_token)
+    :ok = WS.send_binary_frame(peer_ws, auth_request)
+
+    fn ->
+      conn = delete(conn, ~p"/room/#{room_id}/")
+      assert response(conn, :no_content)
+    end
```

Comment:
This seems hacky, maybe let's create `on_exit` callback in `setup` block for the whole file which simply deletes all of the rooms. I considered doing this before but it wasn't necessary back then.

Reply:
Will do

Reply:
Will do